ReportPortal.ATM/WebDriverLibrary/WebDriverLibrary.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Same here, let's disable the implicit usings.

Reply:
Changed to disable